lib/cloudinary/carrier_wave/remote.rb

```diff
@@ -1,5 +1,5 @@
 module Cloudinary::CarrierWave
-  def download!(uri)
+  def download!(uri, _headers = {})
     return super if !self.cloudinary_should_handle_remote?

```

Comment:
This will break on older versions Carrierwave which do not expect an additional param (e.g. `_headers`) when `super` is called.
The solution would be to add `*args` instead of an explicit param.